src/main/java/org/cyclonedx/generators/json/AbstractBomJsonGenerator.java

```diff
@@ -54,6 +56,9 @@ private void setupObjectMapper(final ObjectMapper mapper) {
         SimpleModule depModule = new SimpleModule();
         SimpleModule componentWrapperModule = new SimpleModule();
 
+        mapper.setAnnotationIntrospector(
+            new XmlOnlyAnnotationIntrospector());
```

Comment:
New annotation introspector to exclude fields when generating JSON, that are stuff like the namespace, etc... that show up in XML but not JSON